tools/buf/BUILD.bazel

```diff
@@ -1,17 +0,0 @@
-# Copyright 2021-2022 Buf Technologies, Inc.
```

Comment:
The general approach thats followed is to use a tools directory for the toolchains, so we want to keep this.

Reply:
generally the public API should be in `buf/`

do you want to keep these for backwards compatibility sake?

Reply:
I guess it depends for example `rules_java` has it defined in a similar way: https://github.com/bazelbuild/rules_java/blob/master/toolchains/